modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -341,15 +371,104 @@ dbms.security.oidc.google.token_params=client_secret=GOCSPX-v4cGkygPJvm3Sjjbc0hv
 dbms.security.oidc.google.config=token_type_principal=id_token;token_type_authentication=id_token
 ----
 
-. Log in with your Google SSO credentials using the email address and get the `admin` role when doing so:
-+
-image::sso-configuration-tutorials/oidc-azure-successful-login.png[title="Entra OIDC successful login"]
-+
+There are then two ways to create the user in the database (which is required to give the users roles from native authorization):
+
+. Using Auth Providers.
+xref:authentication-authorization/auth-providers.adoc[Auth Providers] allow us to specify authentication and authorization providers at the user-level which is useful in this scenario.
+This approach relies on the existence of an `admin` user who can authenticate natively, and can themselves create less privileged users who may only authenticate using `oidc-google` and who will receive the roles granted to them using `native` authorization.
+
 [NOTE]
 ====
-The native authentication is disabled to prevent someone from logging in to _alice@neo4j-test.com_ with the set password.
+An admin user with the name `neo4j` is created by default when the database is xref:configuration/set-initial-password.adoc[first started].
 ====
 
+.. Change the configuration to allow `native` authentication (for use only by the `admin` user):
+
+[source, properties]
+----
+dbms.security.authentication_providers=oidc-google, native
+----
+
+.. Set the `dbms.security.require_local_user` configuration setting to `true` in the _neo4j.conf_ file.
```

Comment:
@renetapopova  I've got into a bit of a mess here with way that the ordered list renders... e.g. it keeps resetting its counter because of the interspersed `[source, properties]` elements.

Reply:
> @renetapopova I've got into a bit of a mess here with way that the ordered list renders... e.g. it keeps resetting its counter because of the interspersed `[source, properties]` elements.

No worries. I can fix it when I review the whole PR. Please let me know when it's ready to go ahead.

Reply:
thanks. Yes please, we're ready to go ahead.